lib/controller.js

```diff
@@ -616,6 +616,20 @@ controller.getWifiInfo = function(opts) {
   });
 };
 
+controller.getMemoryInfo = function(opts) {
+  opts.authorized = true;
+  return controller.standardTesselCommand(opts, function(tessel) {
+    return tessel.memoryInfo()
+      .then(function(meminfo) {
+        Object.keys(meminfo).forEach(function(key) {
+          logs.basic(sprintf('\t%s\t%d kB', key, meminfo[key] / 1000));
+        });
+
+        return controller.closeTesselConnections([tessel]);
```

Comment:
You don't need to close the Tessel connections if you use `standardTesselCommand` - it closes the Tessel connection once the provided function resolves. Nice eh? :stuck_out_tongue: